internal/common/server_status.go

```diff
@@ -11,24 +11,33 @@ import (
 	"github.com/edulinq/autograder/internal/util"
 )
 
+type ServerInitiator string
+
 const (
-	PID_SOCK_LOCK                  = "internal.common.PID_SOCK_LOCK"
+	SERVER_STATUS_LOCK             = "internal.common.SERVER_STATUS_LOCK"
 	STATUS_FILENAME                = "status.json"
 	UNIX_SOCKET_RANDNUM_SIZE_BYTES = 32
 )
 
+const (
+	PRIMARY_SERVER  ServerInitiator = "primary-server"
+	CMD_SERVER      ServerInitiator = "cmd-server"
+	CMD_TEST_SERVER ServerInitiator = "cmd-test-server"
+)
```

Comment:
Move this so that it is directly under the type declaration (with a newline between them).
That way, you declare the type and right away enumerate its possible values.